### PR TITLE
Report additions and polish

### PR DESCRIPTION
**Pull request by murphnomer**

### CODE REVIEW

File: Report.md [hunk 1]
```diff
@@ -153,6 +154,15 @@ not involve the composition of previous algorithms. A design from the top down m
 a better perspective on how we needed those algorithms to function from the perspective of
 use with the ML queues.
 
+One aspect of the design process that this project illustrated very well is the advantage of spending time
+analyzing the requirements as a team, discussing available design options, and planning out that design
+before starting to build the solution.  In this case, we ended up with a project that worked, but we had to 
+do some redesign late in the project because we couldn't integrate code that we wrote earlier since we hadn't
+properly planned out how to make the sections work together from the beginning.  Also, although the Process
+class ended up being a useful construct, it is a complex mechanism and not all team members were on the same
+page with it at the start.  This led to a delay in getting some of the alforithms finished while we all worked
```
Comment (jaredscarr) [inline Report.md]: spelling algorithms

File: Report.md [hunk 2]
```diff
@@ -185,42 +195,40 @@ use with the ML queues.
...
+scheduler process itself, for instance). The test data provided contains processes that have priorities assigned to them, 
+and some algorithms take that into consideration.
 
 The graph [Averages by Algorithm](#average-breakdown) above details the averages of each of these criteria broken down
-by algorithm. The FCFS, SJF, and Priority were immediately disqualified due to poor CPU utilization. Which leaves us
-with MLQ, MLFQ, and Round Robin. We then contrasted Round Robin with MLFQ and decided that the lower average wait time
-and response of Round Robin were not significant enough to ignore the 3% bump on CPU utilization it achieved. So we cut
-MLFQ.
+by algorithm. The FCFS, SJF, and Priority were immediately disqualified due to poor CPU utilization, which leaves us
+with MLQ, MLFQ, and Round Robin as candidates for best choice algorithm. We then contrasted Round Robin with MLFQ and
```
Comment (jaredscarr) [inline Report.md]: Not your error but sense you're here. Maybe rewrite "candidates for the best choice of algorithm?" or something that makes more sense. Thanks.

**Review by jaredscarr — CHANGES_REQUESTED** [review]
One spelling and one grammar error of mine I was hoping you could fix.

File: Report.md (hunk 1 above)
Comment (murphnomer) [reply Report.md]: fixed

File: Report.md (hunk 2 above)
Comment (murphnomer) [reply Report.md]: I was thinking of it like "best choice" algorithm here, but I think your suggestion works better anyway, so I used that

**Review by jaredscarr — APPROVED** [review]

**Pull request merged**